pkg/translator/prometheusremotewrite/metrics_to_prw_v2.go

```diff
@@ -33,14 +33,19 @@ func FromMetricsV2(md pmetric.Metrics, settings Settings) (map[string]*writev2.T
 
 // prometheusConverterV2 converts from OTLP to Prometheus write 2.0 format.
 type prometheusConverterV2 struct {
-	// TODO handle conflicts
-	unique      map[uint64]*writev2.TimeSeries
-	symbolTable writev2.SymbolsTable
+	unique map[uint64]*writev2.TimeSeries
+	// conflicts is a map of time series signatures(an unique identifier for TS labels) to a list of TSs with the same signature.
+	// this is used to handle conflicts that occur when multiple TSs have the same labels or when different labels generate the same signature.
```

Comment:
> or when different labels generate the same signature.

I'm struggling a bit to understand how that's possible. The label set is a "signature" of a timeseries, so how can different labels represent the same thing?

Reply:
Oh, right! Would it make sense to mention that in the comment?

Reply:
I think the comment is already clear enough about that, even though it uses the word conflict instead of collision.